modules/graphdb/templates/04_gdb_conf_overrides.sh.tpl

```diff
@@ -32,22 +32,31 @@ aws --cli-connect-timeout 300 ssm get-parameter --region ${region} --name "/${na
 GRAPHDB_CLUSTER_TOKEN="$(aws --cli-connect-timeout 300 ssm get-parameter --region ${region} --name "/${name}/graphdb/cluster_token" --with-decryption | jq -r .Parameter.Value | base64 -d)"
 # Get the NODE_DNS_RECORD value from the previous script
 SSM_PARAMETERS=$(aws ssm describe-parameters --cli-connect-timeout 300 --region ${region} --query "Parameters[?starts_with(Name, '/${name}/graphdb/')].Name" --output text)
+NODE_COUNT=$(aws autoscaling describe-auto-scaling-groups --auto-scaling-group-names ${name} --query "AutoScalingGroups[0].DesiredCapacity" --output text)
 
-cat << EOF > /etc/graphdb/graphdb.properties
+if [ "$NODE_COUNT" -eq 1 ]; then
+  cat << EOF > /etc/graphdb/graphdb.properties
+graphdb.connector.port=7201
+graphdb.external-url=$${PROTOCOL}://$${LB_DNS_RECORD}
+graphdb.external-url.enforce.transactions=true
```

Comment:
Could you add a note/comment somewhere **why** this enforcement is needed? It would be helpful for future maintainers.